packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -374,10 +374,6 @@ Possible alternatives for such technical user authentication are BasicAuthentica
       authHeaderJwt: serviceJwt.encoded, // token to get destination from service
       exchangeHeaderJwt: this.subscriberToken.userJwt.encoded // token considered for user and tenant
     };
-
```

Comment:
why this was removed?

Reply:
Good catch, that was by accident 😅

Reply:
Oh nvm, the reason this was removed, is because it can't be reached (as indicated by my IDE), since there is no continued for the return statement above.

Reply:
oh yes this is correct